README.md

```diff
@@ -3,17 +3,35 @@
 <h1 align="center">Search</h1>
 
 [![Search Plugin license](https://img.shields.io/github/license/monsieurbiz/SyliusSearchPlugin?public)](https://github.com/monsieurbiz/SyliusSearchPlugin/blob/master/LICENSE.txt)
-[![Recipe](https://github.com/monsieurbiz/SyliusSearchPlugin/actions/workflows/recipe.yaml/badge.svg?branch=master&event=push)](https://github.com/monsieurbiz/SyliusSearchPlugin/actions/workflows/recipe.yaml)
-[![Tests](https://github.com/monsieurbiz/SyliusSearchPlugin/actions/workflows/tests.yaml/badge.svg?branch=master&event=push)](https://github.com/monsieurbiz/SyliusSearchPlugin/actions/workflows/tests.yaml)
-[![Security](https://github.com/monsieurbiz/SyliusSearchPlugin/actions/workflows/security.yaml/badge.svg?branch=master&event=push)](https://github.com/monsieurbiz/SyliusSearchPlugin/actions/workflows/security.yaml)
+[![Tests Status](https://img.shields.io/github/actions/workflow/status/monsieurbiz/SyliusSearchPlugin/tests.yaml?branch=master&logo=github)](https://github.com/monsieurbiz/SyliusCmsPagePlugin/actions?query=workflow%3ATests)
+[![Recipe Status](https://img.shields.io/github/actions/workflow/status/monsieurbiz/SyliusSearchPlugin/recipe.yaml?branch=master&label=recipes&logo=github)](https://github.com/monsieurbiz/SyliusCmsPagePlugin/actions?query=workflow%3ASecurity)
+[![Security Status](https://img.shields.io/github/actions/workflow/status/monsieurbiz/SyliusSearchPlugin/security.yaml?branch=master&label=security&logo=github)](https://github.com/monsieurbiz/SyliusCmsPagePlugin/actions?query=workflow%3ASecurity)
+
 
 A search plugin for Sylius using [Elastically](https://github.com/jolicode/elastically) and [Jane](https://github.com/janephp/janephp).
 
+## Compatibility
+
+| Sylius Version | PHP Version |
+|---|---|
+| 1.11 | 8.0 - 8.1 |
+| 1.12 | 8.1 - 8.2 |
+| 1.13 | 8.1 - 8.2 |
+
 ## Installation
 
-Require the plugin :
+If you want to use our recipes, you can configure your composer.json by running:
+
+```bash
+composer config --no-plugins --json extra.symfony.endpoint '["https://api.github.com/repos/monsieurbiz/symfony-recipes/contents/index.json?ref=flex/master","flex://defaults"]'
 ```
-composer require monsieurbiz/sylius-search-plugin="^2"
+
+This is the last version using `jane-php/automapper`, which requires `nikic/php-parser="^4.0"`.
+The next version will use `jolicode/automapper`, which is compatible with `nikic/php-parser="^5.0"`.
+
+```bash
+composer require --no-progress --no-interaction nikic/php-parser="^4.0"
+composer require monsieurbiz/sylius-search-plugin
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** to avoid adding this restriction to composer.json, you can also use the `-W` option

‏
```suggestion
composer require monsieurbiz/sylius-search-plugin -W
```

Reply:
I will try this today to be sure it will not change others packages versions

Reply:
```
--with-dependencies (-w): Update also dependencies of packages in the argument list, except those which are root requirements.
--with-all-dependencies (-W): Update also dependencies of packages in the argument list, including those which are root requirements.
```

With `-w` in lower case I have this diff : 
```
(cd tests/Application && symfony composer require --no-progress --no-interaction monsieurbiz/sylius-search-plugin="*@dev" -w)
./composer.json has been updated
Running composer update monsieurbiz/sylius-search-plugin --with-dependencies
Loading composer repositories with package information
Dependency sylius/sylius is also a root requirement. Package has not been listed as an update argument, so keeping locked at old version. Use --with-all-dependencies (-W) to include root dependencies.
Restricting packages listed in "symfony/symfony" to "^6.4"
Updating dependencies
Lock file operations: 12 installs, 7 updates, 0 removals
  - Locking elasticsearch/elasticsearch (v7.17.2)
  - Locking ezimuel/guzzlestreams (3.1.0)
  - Locking ezimuel/ringphp (1.2.2)
  - Locking jacquesbh/eater (v2.0.4)
  - Locking jane-php/automapper (v7.5.3)
  - Locking jane-php/automapper-bundle (v7.5.3)
  - Locking jolicode/elastically (v1.9.1)
  - Locking monsieurbiz/sylius-search-plugin (dev-master)
  - Locking monsieurbiz/sylius-settings-plugin (v1.2.0)
  - Downgrading nikic/php-parser (v5.1.0 => v4.19.1)
  - Locking nyholm/dsn (2.0.1)
  - Locking react/promise (v2.11.0)
  - Locking ruflin/elastica (7.3.2)
  - Downgrading symfony/cache (v7.1.2 => v6.4.8)
  - Downgrading symfony/clock (v7.1.1 => v6.4.8)
  - Downgrading symfony/error-handler (v7.1.2 => v6.4.9)
  - Downgrading symfony/mime (v7.1.2 => v6.4.9)
  - Downgrading symfony/var-dumper (v7.1.2 => v6.4.9)
  - Downgrading symfony/var-exporter (v7.1.2 => v6.4.9)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 12 installs, 7 updates, 0 removals
  - Downgrading symfony/var-exporter (v7.1.2 => v6.4.9): Extracting archive
  - Installing react/promise (v2.11.0): Extracting archive
  - Installing ezimuel/guzzlestreams (3.1.0): Extracting archive
  - Installing ezimuel/ringphp (1.2.2): Extracting archive
  - Downgrading symfony/var-dumper (v7.1.2 => v6.4.9): Extracting archive
  - Downgrading symfony/error-handler (v7.1.2 => v6.4.9): Extracting archive
  - Downgrading nikic/php-parser (v5.1.0 => v4.19.1): Extracting archive
  - Installing jane-php/automapper (v7.5.3): Extracting archive
  - Downgrading symfony/cache (v7.1.2 => v6.4.8): Extracting archive
  - Downgrading symfony/clock (v7.1.1 => v6.4.8): Extracting archive
  - Downgrading symfony/mime (v7.1.2 => v6.4.9): Extracting archive
  - Installing monsieurbiz/sylius-settings-plugin (v1.2.0): Extracting archive
  - Installing nyholm/dsn (2.0.1): Extracting archive
  - Installing elasticsearch/elasticsearch (v7.17.2): Extracting archive
  - Installing ruflin/elastica (7.3.2): Extracting archive
  - Installing jolicode/elastically (v1.9.1): Extracting archive
  - Installing jane-php/automapper-bundle (v7.5.3): Extracting archive
  - Installing jacquesbh/eater (v2.0.4): Extracting archive
  - Installing monsieurbiz/sylius-search-plugin (dev-master): Symlinking from ../..
```

So I suggest to keep the require at this time with the warning in the README, and we will manage it for next version of the plugin : #214